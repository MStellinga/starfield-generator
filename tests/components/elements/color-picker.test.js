/* eslint-disable jest/valid-expect */
import { h } from 'preact';
import render from 'preact-render-to-string';
import ColorPicker from '../../../src/components/elements/color-picker';

describe('Test color picker', () => {

	test('Color picker renders correctly', () => {
        const tree = render(<ColorPicker color={{r:1,g:2,b:3}} />);
        expect(tree).toMatchSnapshot();	
    });
    
});
