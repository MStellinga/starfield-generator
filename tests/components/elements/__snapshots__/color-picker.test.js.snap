// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test color picker Color picker renders correctly 1`] = `"<div><div class=\\"pickerButton\\" style=\\"background-color: rgb(1,2,3);\\"></div></div>"`;
